optimum/intel/ipex/modeling_base.py

```diff
@@ -171,23 +173,10 @@ def _from_transformers(
         model = TasksManager.get_model_from_task(task, model_id, **model_kwargs)
         traced_model = ipex_jit_trace(model, task, use_cache)
 
-        save_dir = TemporaryDirectory()
-        save_dir_path = Path(save_dir.name)
-        torch.jit.save(traced_model, save_dir_path / WEIGHTS_NAME)
         config.torchscript = True
         config.torch_dtype = torch_dtype
 
-        return cls._from_pretrained(
-            model_id=save_dir_path,
-            config=config,
-            use_auth_token=use_auth_token,
-            revision=revision,
-            force_download=force_download,
-            cache_dir=cache_dir,
-            local_files_only=local_files_only,
-            use_cache=use_cache,
-            model_dtype=torch_dtype,
-        )
+        return cls(traced_model, config=config, model_save_dir=model_id, use_cache=use_cache)
```

Comment:
shouldn't we set `warmup=False` here as some warmup is already done in `ipex_jit_trace` ?

```suggestion
        return cls(traced_model, config=config, model_save_dir=model_id, use_cache=use_cache, warmup=False)
```